types/react-native-material-ui/index.d.ts

```diff
@@ -427,7 +427,7 @@ export interface ToolbarStyle {
 export interface Searchable {
     placeholder?: string | undefined;
     autoFocus?: boolean | undefined;
-    autoCapitalize?: "none" | "sentences" | "words" | "characters" | undefined;
+    autoCapitalize?: "off" | "none" | "on" | "sentences" | "words" | "characters" | undefined;
```

Comment:
Are we sure `react-native-material-ui` supports all these values? If it just forwards this prop, it should just reference the original type instead of inlining it.